### ISSUE TITLE
ILS eis Data 05a en d

### ISSUE THREAD

**Issue by ElisabethDeVries** [Stichting-CROW/PCB#34]
E. van O. uit Zuid-Holland vraagt: 
Eerste vraag is waarom deze eisen opgesplitst zijn. Is de informatie voor de WIBON anders dan wat een organisatie verder nog zou willen weten van de ondergrondse infra?

**Comment by ElisabethKloren** [comment]
Data 05a en d zijn twee verschillende formaten gegevens aangeleverd kan krijgen. 05a –in een eenvoudig formaat, in 05b in linked data. 
In beide gevallen gaan de eisen over de gegevens voor de wibon en aanvullingen daarop, bijvoorbeeld informatie die je in je beheersysteem wilt hebben (uit IMBOR), de tekst is dus niet helemaal correct als ik het zo lees; voorstel voor aanpassing  na (WIBON) "en de aanvullende gegevens nodig voor beheer"

**Comment by ElisabethKloren** [comment]
E. van O. uit Zuid-Holland vraagt:
Tweede vraag, En is een format Wibon per organisatie anders?

**Comment by ElisabethKloren** [comment]
Ja, juist omdat je ook eigen aanvullingen kan doen – de titel dekt mogelijk niet helemaal de lading: tiltel aan te passen
